script/metalsmith/breadcrumbTransformer.js

```diff
@@ -0,0 +1,26 @@
+const cheerio = require('cheerio');
+const fs = require('fs');
+const path = require('path');
+const UglifyJS = require('uglify-js');
+
+const breadcrumb = fs.readFileSync(path.join(__dirname, '../../assets/js/breadcrumbs/mobile-breadcrumb.js'), 'utf8');
+
+module.exports = (files, metalsmith, done) => {
+  Object.keys(files).forEach(file => {
+    if (path.extname(file) !== '.html') return;
+
+    const data = files[file];
+    const $ = cheerio.load(data.contents.toString());
+    const minifiedCode = UglifyJS.minify(breadcrumb);
```

Comment:
Might be useful to minify this outside of the `forEach` loop, since the same minified code is used for each file.